assets/scss/core/_typography.scss

```diff
@@ -126,4 +129,4 @@ small {
   font-weight: 400;
   font-style: normal;
 }
-@import url('https://fonts.googleapis.com/css?family=Merriweather');
\ No newline at end of file
+// @import url('https://fonts.googleapis.com/css?family=Merriweather');
```

Comment:
Why did you keep this line commented?

Reply:
Hice una minilimpieza de los estilos y agregue cosas que faltaban. Esta linea se puede borrar sin problemas.

Reply:
No estaba seguro de que se utilizaba y que no. Se me paso borrarla. Se elimina sin problemas. No podemos tener enlaces externos dentro de los styles